api/src/pcapi/core/offers/api.py

```diff
@@ -1960,14 +1960,24 @@ def move_event_offer(
         transactional_mails.send_email_for_each_ongoing_booking(offer)
 
 
-def update_used_stock_price(stock: models.Stock, new_price: float) -> None:
+def update_used_stock_price(
+    stock: models.Stock, new_price: float | None = None, price_percent: decimal.Decimal | None = None
+) -> None:
     if not stock.offer.isEvent:
         raise ValueError("Only stocks associated with an event offer can be edited with used bookings")
+    if new_price and price_percent or not (new_price or price_percent):
```

Comment:
si c'est un `xor`, je propose de simplifier pour gérer aussi les cas de `new_price == 0` : `if (new_price is None) == (price_percent is None):`

Reply:
d'un point de vue logique je suis d'accord mais je trouve ça moins lisible